deploy/chart/values-external.yaml

```diff
@@ -0,0 +1,65 @@
+# Values for register clusters
+# NOTE: on-cloud nodes created by register clusters will have alibabacloud.com/external=true label
+
+deploy:
+  accessKey:
+    enabled: true
+    secretName: alibaba-addon-secret
+    idField: access-key-id
+    secretField: access-key-secret
```

Comment:
Maybe we should also add yet another value `deploy.addonToken`, default to `deploy.ack`, and override it to false on external cluster.

Reply:
Or just disable addonToken automatically if accessKey is enabled?

Reply:
I don't think there is a problem, because the addon token volume is set as optional. csi will prioritize to use credentials from env variables.